cmd/start/main.go

```diff
@@ -121,11 +126,42 @@ func start(c *cobra.Command, args []string) {
 		"default-tempo-query-image", rootCmdConfig.CtrlConfig.DefaultImages.TempoQuery,
 		"default-tempo-gateway-image", rootCmdConfig.CtrlConfig.DefaultImages.TempoGateway,
 		"default-tempo-gateway-opa-image", rootCmdConfig.CtrlConfig.DefaultImages.TempoGatewayOpa,
+		"default-network-policies", ctrlConfig.Gates.NetworkPolicies,
 		"go-version", version.GoVersion,
 		"go-arch", runtime.GOARCH,
 		"go-os", runtime.GOOS,
 	)
 
+	if ctrlConfig.Gates.NetworkPolicies {
+		objs, err := networking.GenerateOperatorPolicies()
```

Comment:
Can you merge this with the reconcile of the operator?
https://github.com/grafana/tempo-operator/blob/7a36953a4228b039b622745c2c916a9b45dda4a9/internal/controller/tempo/operator_controller.go#L53

Manifests:
https://github.com/grafana/tempo-operator/blob/7a36953a4228b039b622745c2c916a9b45dda4a9/internal/manifests/operator/manifests.go#L10-L12